packages/edit-site/src/store/reducer.js

```diff
@@ -98,10 +98,20 @@ function editorCanvasContainerView( state = undefined, action ) {
 	return state;
 }
 
+function routes( state = [], action ) {
+	switch ( action.type ) {
+		case 'REGISTER_ROUTE':
+			return [ ...state, action.route ];
```

Comment:
We should probably also have a method to register multiple routes at once. Instead of having to go through `registry.batch`. Plugins will use it all the time.

Do we ever need to unregister a route?